WheelOfFortune/Puzzle.cs

```diff
@@ -135,22 +162,16 @@ public bool IsLetterGuessedBefore(char letter)
         /// <param name="letter">existing passed letter</param>
         public void UpdatePuzzleSoFar(char letter)
         {
-            var updatedPuzzleSoFar = new StringBuilder(); ;
-            for (int index = 0; index < PuzzleAnswer.Length; index++)
+            if (PuzzleDictionary.ContainsKey(letter))
```

Comment:
I don't think this check is needed because another function checks this logic before this function is called

Reply:
Updated!